admin/database/postgres/postgres.go

```diff
@@ -210,6 +210,17 @@ func (c *connection) FindProjects(ctx context.Context, afterName string, limit i
 	return res, nil
 }
 
+func (c *connection) FindProjectNamesByPattern(ctx context.Context, namePattern string) ([]*database.ProjectNames, error) {
+	var res []*database.ProjectNames
+	wildcard := "%"
+	pattern := fmt.Sprintf("%s%s%s", wildcard, namePattern, wildcard)
```

Comment:
Remove the wildcard here – it should be provided by the user at the CLI if they want it (then they can do a prefix search if they want by passing `rilldata/%`